browser-test/src/header.test.ts

```diff
@@ -1,56 +1,59 @@
-import {test, expect} from '@playwright/test'
+import {test, expect} from './support/civiform_fixtures'
 import {
-  createTestContext,
   loginAsTestUser,
   validateScreenshot,
   validateAccessibility,
+  logout,
```

Comment:
long term these individual methods will get grouped into logical fixtures of their own